tools/flash_shelly.py

```diff
@@ -346,9 +360,12 @@ def parse_info(device_info, action, dry_run, silent_run, mode, exclude, version,
         homekit_dlurl = f'http://rojer.me/files/shelly/{version}/shelly-homekit-{model}.zip'
       break
 
-  stock_lfw = stock_release_info['data'][device_info['stock_model']]['version'].split('/v')[1].split('@')[0]
+  stock_lfw = stock_release_info['data'][stock_model]['version'].split('/v')[1].split('@')[0]
```

Comment:
do not repeat yourself (DRY):

```
stock_model_info = stock_release_info['data'][stock_model]
# Give example of the version string you're expecting, it's hard to follow parsing code below.
stock_lfw = stock_model_info['version'].split('/v')[1].split('@')[0]

if not version:
  stock_dlurl = stock_model_info['url']
  if device_info['stock_model']  == 'SHRGBW2':
    stock_dlurl = stock_dlurl.replace('.zip', '-'+colour_mode+'.zip')
```

Reply:
ok, this should be cleaner https://github.com/mongoose-os-apps/shelly-homekit/pull/292/commits/2f287f4f89c4e5c68b2923fe50b65bb54e9ecb3d